code/obj/item/implant.dm

```diff
@@ -855,6 +856,54 @@ THROWING DARTS
 		icon = 'icons/obj/chemical.dmi'
 		desc = "A small hollow dart."
 		icon_state = "syringeproj"
+		bleed_time = 0
+		var/barbed = FALSE
+
+		tranq_dart_sleepy
+			name = "tranquilizer dart"
+			desc = "A small tranquilizer dart, emptied of its contents. Useful for putting animals (or people!) to sleep."
+			icon_state = "tranqdart_red"
+
+			New()
+				..()
+				implant_overlay = image(icon = 'icons/mob/human.dmi', icon_state = "tranqdart_red_stick_[rand(0, 4)]", layer = MOB_EFFECT_LAYER)
+
+		tranq_dart_sleepy_barbed
+			name = "barbed tranquilizer dart"
+			desc = "An empty tranquilizer dart, with a barbed tip. It was likely loaded with some bad stuff..."
+			icon_state = "tranqdart_red_barbed"
+			barbed = TRUE
+
+			New()
+				..()
+				implant_overlay = image(icon = 'icons/mob/human.dmi', icon_state = "tranqdart_red_stick_[rand(0, 4)]", layer = MOB_EFFECT_LAYER)
+
+		tranq_dart_mutadone
+			name = "tranquilizer dart"
+			desc = "A small tranquilizer dart, emptied of its contents. This one is specialized for removing genetic mutations."
+			icon_state = "tranqdart_green"
+
+			New()
+				..()
+				implant_overlay = image(icon = 'icons/mob/human.dmi', icon_state = "tranqdart_green_stick_[rand(0, 4)]", layer = MOB_EFFECT_LAYER)
+
+	syringe
```

Comment:
To a newbie it might be confusing why they can't use the 'syringe' they just found on the floor- a more descriptive name like "spent syringe projectile" or "spent syringe round" would be nice

Reply:
I didn't put that on the barbed ones, felt maybe it was a bit wordy saying something like spent barbed tranquilizer dart/syringe, let me know if I should. I tried to add a bit more to the descriptions for those to make up for it.